src/app.js

```diff
@@ -215,48 +217,134 @@ export default function (app) {
       var realValue = element[name]
 
       if (name === "onUpdate") {
-        defer(value, element)
+          defer(value, element)
 
       } else if (value !== oldValue || realValue !== value) {
         setElementData(element, name, value, oldValue)
       }
     }
   }
 
-  function patch(parent, element, oldNode, node) {
-    if (oldNode === undefined) {
-      element = parent.appendChild(createElementFrom(node))
-
-    } else if (node === undefined) {
+  function batchRemove (parent, element, node) {
```

Comment:
I don't se any problem batching the onRemove-handler, BUT even before this PR, the onRemove handler was deferred, meaning it gets was called *after* the node is removed. So... do you want me to fix that behavior as well, or leave it as it is?